twbt.cpp

```diff
@@ -144,7 +145,7 @@ long white_texpos, transparent_texpos;
 
 long cursor_small_texpos;
 
-static bool enabled;
+DFHACK_PLUGIN_IS_ENABLED(enabled);
```

Comment:
Will this have any side-effects since enabling/disabling isn't actually supported?